repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -121,6 +124,7 @@
                     <xs:element name="artifact-directory" type="papic:ArtifactDirectory" minOccurs="1" maxOccurs="1"/>
                     <xs:element name="ssl-configuration" type="papic:SslConfiguration" minOccurs="0" maxOccurs="1"/>
                     <xs:element name="logging-configuration" type="papic:LoggingConfiguration" minOccurs="0" maxOccurs="1"/>
+                    <xs:element name="via-configuration" type="papic:ViaConfiguration" minOccurs="0" maxOccurs="1"/>
```

Comment:
I was just using the naming convention used in this particular configuration file. I good with calling it `via-header` if that is what we want to call it. A name is a name to me unless there is some compelling reason for it not to be named something. I do not feel strongly either way on this one, so I will leave that for the team to decide.